Quaver.Shared/Screens/Gameplay/Rulesets/Keys/Playfield/GameplayPlayfieldKeysStage.cs

```diff
@@ -565,7 +565,7 @@ private void CreateJudgementHitBurst()
                 {
                     Parent = Playfield.ForegroundContainer,
                     Alignment = Alignment.MidCenter,
-                    X = skin.DisplayJudgementsInEachColumn ? Receptors[lane].X - playfieldOffset : 0
+                    X = skin.DisplayJudgementsInEachColumn ? Receptors[lane].X - playfieldOffset : (skin.JudgementBurstPosX)
```

Comment:
Nitpick: I'd rather enclose in () the expression to the left of : as it is multi-part, rather than this.

Reply:
You're right, I'll fix it